internal/storage/rocksdb/store.go

```diff
@@ -531,6 +618,15 @@ func (rdb *rocksDB) getMultipleKeys(ro *gorocksdb.ReadOptions, keys [][]byte) ([
 	for i, value := range values {
 		if value != nil {
 			key, val := keys[i], toByteArray(value)
+			//check ttl.
+			expireTs, _ := getExpireTs(val)
+			if expireTs > 0 && expireTs < time.Now().Unix() {
+				//triger delete.
+				rdb.Delete(key)
```

Comment:
Same as my earlier comment on the need for deletion.

Reply:
Done